docs/md/staged-workload.md

```diff
@@ -25,21 +25,11 @@ The specific values below are from the
 [COVID-19 Surveillance in Terra](./modules-covid.md) project.
 Workloads for other projects may leverage different implementations for source, executor or sink.
 
-!!! tip "`watchers` of a workload"
-    You may have noticed the optional `watchers` field in the requests.
-    As of `2021/08` WFL supports registering Slack channels
-    (Slack channel IDs that starts with a `C`)
-    as watchers of the workload.
-    Your channel needs to live in the `broadinstitute.slack.com` Slack organization
-    and you also have to `/invite @WorkFlow Launcher Notifier`
-    to invite the WFL notifier to your channel
-    before the channel can receive user facing messaged from WFL.
-
 ```
 {
     "watchers": [
         ["slack", "C000XXX0XXX"],
-        ["email", "tester@broadinstitute.org"]
```

Comment:
I removed all traces of emails as watchers from these docs.  They're misleading, since we don't do anything with them.